level-1/자연수-뒤집어-배열로-만들기&12932&.js

```diff
@@ -64,3 +64,11 @@ function solution(n) {
   }
   return answer;
 }
+
+// 정답 7 - inhwa-jang
+function solution(n) {
+    let str = String(n);
+    let newArr = Array.from(str);
+    newArr.reverse();
+    return newArr.map(Number);
```

Comment:
아래와 같은 메서드 체이닝을 도입해 가독성을 높혀보세요!
```suggestion
    return Array.from(String(n)).reverse().map(Number);
```